### PR TITLE
Added Pull Request Template

### LINKED ISSUE #91: [DOCS] Addition of Pull Request Template
### Description

Addition of Pull Request Template to improve clarity and to make it easier for new contributors.

*Please assign this issue to me under GSSoC'23* @J0SAL 

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by DivyanshiNarang**
Resolves #91

Created Pull Request Template so it is easier for maintainers of the project to review PR and make sure that no PR is submitted without proper description.

### CODE REVIEW

**Comment by DivyanshiNarang** [comment]
@J0SAL, please review it.

**Comment by J0SAL** [comment]
@Abhijit25Mishra can you review this

**Comment by Abhijit25Mishra** [comment]
Sure

**Comment by Abhijit25Mishra** [comment]
- Make your PR title specific (if possible)
- Update your PR template like this:
> 
> Fixes: #99
> Description:
> Attach Screenshots (if needed)
>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- Pull Request Template -->
+
+## Related Issue
```
Comment (Abhijit25Mishra) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: change this to something like

## Insert PR title

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Resolves #issue_number
```
Comment (Abhijit25Mishra) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: change this from resolves to fixes:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+## Description
+
+<!-- Write a brief description of the changes made in the PR. Explain the problem being addressed, or any relevant
+information. -->
+
+## Screenshots
+
+<!-- Add screenshots to preview the changes  -->
+
+## Checklist
```
Comment (Abhijit25Mishra) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: I think all the points dont make much sense. its obvious most of the time to do this. atleast for anyone who isnt contributing for the first time. 

So its better to remove. @J0SAL Please let us know of your thoughts on this.

**Review by Abhijit25Mishra — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (J0SAL) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: No, the points make good sense.
A lot of beginners/participants just create a PR for the sake of it, this structure will help provide some formality while creating a PR

Comment (DivyanshiNarang) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: Sure, I will make the required changes.

**Comment by J0SAL** [comment]
@DivyanshiNarang Looks good, Thanks!

**Pull request merged**